Nexus.Sdk.Token/TokenServerProvider.cs

```diff
@@ -6,6 +6,7 @@
 using Nexus.Sdk.Shared.Responses;
 using Nexus.Sdk.Token.Requests;
 using Nexus.Sdk.Token.Responses;
+using System.Security.AccessControl;
```

Comment:
This namespace?

Reply:
Thank you for spotting that one, I should have picked it up before sending to review.

I removed it :)